### PR TITLE
Update docs Rubygems version to beta

### LINKED ISSUE #23292: Change alpha to beta for Rubygems
https://rubygems.org/gems/bootstrap/versions/4.0.0.beta

### PR DESCRIPTION
**Pull request by kukac7**
Fixes #23292

### CODE REVIEW

File: docs/4.0/getting-started/download.md [hunk 1]
```diff
@@ -68,13 +68,13 @@ Bootstrap's `package.json` contains some additional metadata under the following
 Install Bootstrap in your Ruby apps using [Bundler](https://bundler.io/) (**recommended**) and [RubyGems](https://rubygems.org/) by adding the following line to your [`Gemfile`](https://bundler.io/gemfile.html):
 
 {% highlight ruby %}
-gem 'bootstrap', '~> 4.0.0.alpha6'
+gem 'bootstrap', '~> 4.0.0.beta'
```
Comment (Johann-S) [inline docs/4.0/getting-started/download.md]: Isn't it beta1 instead of beta ?

Comment (kukac7) [inline docs/4.0/getting-started/download.md]: here it is: `gem 'bootstrap', '~> 4.0.0.beta'`
https://github.com/twbs/bootstrap-rubygem/blob/master/README.md

**Review by XhmikosR — CHANGES_REQUESTED** [review]
@kukac7: please update the version in index.html too.

**Comment by kukac7** [comment]
@XhmikosR https://github.com/twbs/bootstrap/pull/23300

**Comment by XhmikosR** [comment]
Sorry that is just wrong. Push here.

**Review by XhmikosR — APPROVED** [review]

**Comment by XhmikosR** [comment]
I went ahead and did it manually, but it would be nice if you got the gist of how things work for future contributions.

**Pull request merged**